JS/studentRequest.js

```diff
@@ -53,145 +50,134 @@ async function GetAllRequests(){
-    var statusFilter = document.getElementById("statusFilter").value;
-
-    // Get the table and rows
-    var table = document.getElementById("StudentRequestTable");
-    var rows = table.getElementsByTagName("tr");
-
-    // Loop through all table rows, hide those that don't match the filter criteria
-    for (var i = 1; i < rows.length; i++) {
-        var row = rows[i];
-        var university = row.cells[2].textContent;
-        var status = row.cells[5].textContent;
-
-        var universityMatch = universityFilter === '' || university === universityFilter;
-        var statusMatch = statusFilter === '' || status === statusFilter;
-
-        if (universityMatch && statusMatch) {
-            row.style.display = "";  // Show the row
-        } else {
-            row.style.display = "none";  // Hide the row
-        }
+  // Get selected values from filters
+  let universityFilter = document.getElementById("universityFilter").value;
+  let statusFilter = document.getElementById("statusFilter").value;
+
+  // Get the table and rows
+  let table = document.getElementById("StudentRequestTable");
+  let rows = table.getElementsByTagName("tr");
+
+  // Loop through all table rows, hide those that don't match the filter criteria
+  for (let i = 1; i < rows.length; i++) {
+    let row = rows[i];
+    let university = row.cells[2].textContent;
+    let status = row.cells[5].textContent;
+
+    let universityMatch =
+      universityFilter === "" || university === universityFilter;
+    let statusMatch = statusFilter === "" || status === statusFilter;
+
+    if (universityMatch && statusMatch) {
+      row.style.display = ""; // Show the row
+    } else {
+      row.style.display = "none"; // Hide the row
     }
+  }
 }
 
 function populateFilterOptions() {
-    var universityFilter = document.getElementById("universityFilter");
-    var statusFilter = document.getElementById("statusFilter");
-
-    var universityValues = getUniqueColumnValues(document.getElementById("StudentRequestTable"), 2); // Assuming university is in the third column
-    var statusValues = getUniqueColumnValues(document.getElementById("StudentRequestTable"), 5); // Assuming status is in the sixth column
-
-    populateDropdown(universityFilter, universityValues);
-    populateDropdown(statusFilter, statusValues);
+  let universityFilter = document.getElementById("universityFilter");
+  let statusFilter = document.getElementById("statusFilter");
+
+  let universityValues = getUniqueColumnValues(
+    document.getElementById("StudentRequestTable"),
+    2
+  ); // Assuming university is in the third column
+  let statusValues = getUniqueColumnValues(
+    document.getElementById("StudentRequestTable"),
+    5
+  ); // Assuming status is in the sixth column
+
+  populateDropdown(universityFilter, universityValues);
+  populateDropdown(statusFilter, statusValues);
 }
 
 // Helper function to populate a dropdown with options
 function populateDropdown(selectElement, values) {
```

Comment:
I was thinking if possible we find a way to store helper functions in on file then we re-use then wherever we need them.